docroot/sites/all/themes/ilr_theme/scss/components/_view_modes.scss

```diff
@@ -134,20 +134,22 @@
   }
 
   body:not(.mobilizing-against-inequality) & {//don't display as card on MAI home
-    background: white;
-    border: 2px solid #e0e0e0;
-    border-bottom-width: 6px;
+    border: 1px solid $gray-light;
+    border-radius: 6px;
     @include trailer(1.5);
     @include padding-leader(1);
+    &:hover {
+      background: white;
+      border-color: $gray-medium;
```

Comment:
Probably. Wouldn't make sense to include a border that people cannot see. I can switch the color to another gray that I have been using on other elements that is quite visible.